Java/String/JavaAnagrams.java

```diff
@@ -0,0 +1,53 @@
+// https://www.hackerrank.com/challenges/java-anagrams/problem
+
+import java.util.*;
+
+public class Solution {
+    static boolean isAnagram(String a, String b) { 
+        char array1[] = a.toLowerCase().toCharArray();
+        char array2[] = b.toLowerCase().toCharArray();
+        
+        sortArray(array1);
+        sortArray(array2);
+
+        if (array1.length != array2.length) {
+            return false;
+        } else {
+            int check = 0;

```

Comment:
Use a `boolean` variable instead of `int`. We don't need to check how many times it's wrong.

A better way is not using any variable.

For example:

    Run a loop
    - If there is any non-matching character, return `false`.
    - Loop is run completely, return `true`.